### DISCUSSION TITLE
Do we need a place in a competency framework to identify if it is a team/group KSA?

### DISCUSSION THREAD

**Discussion by philbarker** [CredentialEngine/Schema-Development#759] [Ideas]
As mentioned in our Feb 23 2021 CTDL meeting, the W3C VC Ed group discussed groups as the subject of a verfiable credential, e.g. what does it mean to say that a group has a competency. They decided to park the issue, focussing on natural persons as credential subjects (holders), however it's worth considering how we would deal with this, that is:  do we need a place in a competency framework to identify if it is a team/group KSA?  For example: military assigns billets based on team-based competencies. 
Questions:
- are we talking about competencies that relate to an individual's perfomance within a team setting, or
- are we talking about competencies that must be held by all members of a team, or
- are we talking about competencies of the team as a whole (being greater than the sum of its parts) ?

- how would team/group competencies relate to organizational competencies, and accreditation (e.g. ISO 9001) ?

**Comment by stuartasutton** [comment]
Phil, do you think we might experiment here in github by moving this
"issue" to a "discussion"...because I think there will likely be a lot over
a period of time :-)?  I see there is a link on the issue page to "transfer
discussion".

**Reply by philbarker** [reply]
@stuartasutton this is a discussion :)

**Reply by philbarker** [reply]
ps: the transfer discussion link is to transfer to a different repo.

**Reply by stuartasutton** [reply]
Getting old sucks...

**Reply by philbarker** [reply]
> Getting old sucks...

But it beats the alternative

**Comment by siuc-nate** [comment]
Interesting suggestion. A couple of things come to mind:
- What would be the use case/benefit for identifying a competency as a group competency?
- Can this be solved by an existing property (e.g. ceasn:competencyCategory or ceasn:conceptTerm) given the limited extent of its use/relevance?
  - In other words, should we wait until there's more of a demand for it?
- What about competencies that could apply both to an individual and/or to a group (e.g. "can fix a car")?
- In our Navy work, there was an area of the source schemas (GEIA0007/S3000L) that defined the **Task** as being either an individual or a group thing.  Are "group competencies" really just "group tasks"?
  - Also worth noting that the parallel/upcoming work had to do with tracking individuals (rather than groups, at least as far as I'm aware), which would imply "tracking individuals within a team setting"
